glue/viewers/profile/state.py

```diff
@@ -230,6 +230,10 @@ def _reset_y_limits(self, *event):
                 if y_max > y_min:
                     self.y_min = y_min
                     self.y_max = y_max
+                elif np.allclose(y_min, y_max):
+                    dy = max(0.1, 0.1 * y_min)
+                    self.y_min = y_min - dy
+                    self.y_max = y_max + dy
```

Comment:
```suggestion
                    self.y_min = y_min * 0.9
                    self.y_max = y_max * 1.1
```
Setting a minimum range of ±0.1 seems quite arbitrary here and would lead to the data values basically appearing as zero if they do not happen to be all 1 but something much smaller.
Could in principle even check for `y_min == y_max` here, since by construction it cannot be larger, and the only other possible case is both still being `np.inf`, which is covered in the `else`.

Reply:
I don't understand your objection. My code isn't +/- 0.1 absolute. It is at minimum 0.1 (say, if the data is zeroes) but otherwise 10 percent of the value (which scales). Your suggested change does not guard against data that is all zeroes.

Reply:
Yes, but if it's all << 0.1, it does not scale. E.g. setting the data in you example to `1e-6 * np.ones((7, 8, 9))` with the current version it is displayed as
<img width="1121" alt="max(0.1, 0.1 * y_min)" src="https://github.com/user-attachments/assets/51a186fe-8635-4b57-be13-e59c380d0d36">
and having all negative data (–1) as
<img width="1122" alt="flux[:] == –1" src="https://github.com/user-attachments/assets/adb248ee-071a-489d-8444-a55e51053f20">
while with my suggestion it looks like
<img width="1120" alt="0.1 * y_min" src="https://github.com/user-attachments/assets/4854f6d2-0cab-432b-be20-194935d56655">
though for negative values the scale is inverted, and the all-zeros case indeed needs to be addressed separately.